### ISSUE TITLE
error - ReferenceError: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by bozzmob** [timlrx/tailwind-nextjs-starter-blog#377]
**Describe the bug**
`yarn start` throws an error `error - ReferenceError: regeneratorRuntime is not defined`

**To Reproduce**
1. Clone repo
2. `yarn install`
3. `yarn start`

**Expected behavior**
The blog should load on `http://localhost:3000/`

**Screenshots**

**System Info (if dev / build issue):**
 - OS: Windows 11
 - Node version: 16.13.1
 - Npm version: 8.4.1

```
tailwind-nextjs-starter-blog> yarn start
yarn run v1.22.15
$ cross-env SOCKET=true node ./scripts/next-remote-watch.js ./data

warn - You have enabled experimental features: optimizeUniversalDefaults
warn - Experimental features in Tailwind CSS are not covered by semver, may introduce breaking changes, and can change at any time.
event - compiled client and server successfully in 12.6s (254 modules)
> Ready on http://localhost:3000
wait  - compiling / (client and server)...
event - compiled client and server successfully in 1336 ms (295 modules)
error - unhandledRejection: ReferenceError: regeneratorRuntime is not defined
wait  - compiling /_error (client and server)...
event - compiled client and server successfully in 121 ms (296 modules)
error - ReferenceError: regeneratorRuntime is not defined
```

**Comment by timlrx** [comment]
Looks like it is due to an error in the latest version of `rehype-citation`. As a workaround, you can pin it to v0.2.0 first or remove it if you do not need the citation feature.

**Comment by timlrx** [comment]
This is fixed in v0.2.4 so you can update to the latest version as well.

**Comment by bozzmob** [comment]
Thanks @timlrx for the quick fix.
